index.html

```diff
@@ -10,7 +10,12 @@
 </head>
 <body>
 
-    <h1>Hello world!!!</h1>
+    <div class="container">
+        <p> ime grada</p>
```

Comment:
Delete this element because you will need just an input with the placeholder "Please enter your place here..."

Reply:
I fix that